README.rst

```diff
@@ -6,17 +6,40 @@ A simple asynchronous OPC UA plugin that registers for change events on OPC UA o
 
 NOTE:
 
-Thia plugin assumes the freeopcua is available at a fixed location in the file system. To build you
-must clone the freeopcua repository in your home directory.
+This plugin assumes the freeopcua is available at a specified location in the file system. To build you
+must clone the freeopcua repository to a directory of your choice.
 
 .. code-block:: console
 
   $ git clone https://github.com/FreeOpcUa/freeopcua.git
+  $ cd freeopcua
+  $ export OPCUADIR=`pwd`
```

Comment:
As per the script `set(OPCUADIR $ENV{FREEOPCUA})` it should be `export FREEOPCUA=...`